examples/src/main/java/org/apache/spark/examples/sql/JavaSQLDataSourceExample.java

```diff
@@ -139,11 +139,9 @@ private static void runBasicParquetExample(SparkSession spark) {
     // Parquet files can also be used to create a temporary view and then used in SQL statements
     parquetFileDF.createOrReplaceTempView("parquetFile");
     Dataset<Row> namesDF = spark.sql("SELECT name FROM parquetFile WHERE age BETWEEN 13 AND 19");
-    Dataset<String> namesDS = namesDF.map(new MapFunction<Row, String>() {
-      public String call(Row row) {
-        return "Name: " + row.getString(0);
-      }
-    }, Encoders.STRING());
+    Dataset<String> namesDS = namesDF.map(
+        (MapFunction<Row, String>) row -> "Name: " + row.getString(0),
```

Comment:
For those wondering why this lambda has to be cast, it's because it is otherwise ambiguous, as there are two methods with an eligible function interface as arg.